src/gui/wizard/owncloudsetuppage.cpp

```diff
@@ -127,7 +127,11 @@ void OwncloudSetupPage::slotLogin()
 void OwncloudSetupPage::slotGotoProviderList()
 {
     _ocWizard->setRegistration(true);
+#ifdef WITH_WEBENGINE
```

Comment:
I am pretty sure this is wrong here as you are forcing OAuth instead of Web Flow V2